h/static/scripts/login-forms/config.ts

```diff
@@ -1,7 +1,13 @@
 import { createContext } from 'preact';
 
+export type FlashMessage = {
+  type: 'success' | 'error';
+  message: string;
+};
```

Comment:
This structure is chosen to map closely to what the `ToastMessages` component uses, simplifying the frontend logic.